include/kafka/AdminClient.h

```diff
@@ -292,5 +323,51 @@ AdminClient::listTopics(std::chrono::milliseconds timeout)
     return Admin::ListTopicsResult(names);
 }
 
+inline Admin::DeleteRecordsResult
+AdminClient::deleteRecords(const TopicPartitionOffsets& tpos,
```

Comment:
`topicPartitionOffsets` should be better than `tpos`

Reply:
Fixed like above